ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -129,10 +128,8 @@ Item {
               
             draggableItems: root.store.chatsModelInst.communities.activeCommunity.admin
             draggableCategories: root.store.chatsModelInst.communities.activeCommunity.admin
-            chatList.model: root.store.chatsModelInst.communities.activeCommunity.chats
-
-            categoryList.model: root.store.chatsModelInst.communities.activeCommunity.categories
-
+            chatList.model: store.model
+            categoryList.model: store.model.subItems
```

Comment:
I know this is not scope for this PR, but I just want to leave this here:

The fact that the new model is called `model.subItems` makes it very hard to know what data one is dealing with here. I think it'd be better if we made model expressions more descriptive.

Reply:
yes, this will change with the latest updates in [StatusQ](https://github.com/status-im/StatusQ/pull/486)